src/views/connected/structure/candidatures/ButtonsAction.js

```diff
@@ -10,44 +10,32 @@ import { scrollTopWindow } from '../../../../utils/exportsUtils';
 
 //Print datePicker calendar in FR
 registerLocale('fr', fr);
-function ButtonsAction({ statut, updateStatut, miseEnRelationId, dateRecrutement, dateRupture, motifRupture }) {
+function ButtonsAction({ statut, updateStatut, miseEnRelationId, dateRecrutement }) {
 
   const dispatch = useDispatch();
 
   const [dateValidee, setDateValidee] = useState(dateRecrutement);
-  const [dateRuptureValidee, setDateRuptureValidee] = useState(dateRupture);
-  const [motifRuptureValide, setMotifRuptureValide] = useState(motifRupture);
   const [openModal, setOpenModal] = useState(false);
-  const today = new Date();
 
   const updateDateRecrutement = date => {
     scrollTopWindow(); //permet de remonter pour visualiser le message date embauche enregistrée
     date = dayjs(date);
     dispatch(conseillerActions.updateDateRecrutement({ id: miseEnRelationId, date }));
   };
 
-  const updateDateRupture = date => {
-    date = dayjs(date);
-    dispatch(conseillerActions.updateDateRupture({ id: miseEnRelationId, date }));
-  };
-
-  const updateMotifRupture = motif => {
-    dispatch(conseillerActions.updateMotifRupture({ id: miseEnRelationId, motif }));
-  };
-
   return (
     <div className="">
       <div className="fr-grid-row">
         {statut === 'nouvelle' &&
           <div className="fr-col-5">
-            <button onClick={updateStatut.bind(this, 'interessee')} className="fr-btn fr-icon-success-line fr-btn--icon-left" title="Pré sélectionner">
+            <button onClick={() => updateStatut('interessee')} className="fr-btn fr-icon-success-line fr-btn--icon-left" title="Pré sélectionner">
```

Comment:
ca ne marchait plus ?

Reply:
dans les composants fonctionnel on a plus besoin du 'this' contrairement au composant de class